src/chart/graph/GraphView.ts

```diff
@@ -287,6 +293,7 @@ class GraphView extends ChartView {
                 this._lineDraw.updateLayout();
                 // Only update label layout on zoom
                 api.updateLabelLayout();
+                this._thumbanil._updateZoom(e);
```

Comment:
No need to separate `zoom` and `pan` update. We need to always recaculate the selected rect whenever `zoom`, `pan` or `re-render`